### DISCUSSION TITLE
failed on ota install macOS Ventura 13.0

### DISCUSSION THREAD

**Discussion by HenryChanhy** [osy86/HaC-Mini#1035] [Q&A] [answered]
When upgrade reboot. It stucked at oc: "configuration requires signed vault but no public key provided" 
I had try to fresh install ventura or recovery but nothing work.
What can I do to fix this? Thanks for any help.

before OTA upgrade:
Hacmini 3.2.0 on NUC8i7HVK macOS monterey

**Comment by osy** [comment]
Did you read and follow the advise on the release notes about secure boot?

**Reply by HenryChanhy** [reply]
Sure! The bless command got an err:
Personalization failed: Error Domain=OSPErrorDomain Code=200 "The signing server declined the personalization request." UserInfo={NSLocalizedDescription=The signing server declined the personalization request., preductionStatus=true,NSUnderlyingError=0x600003a401e0 {Error Domain=AuthInstallErrorDomain Code=3194 "(null)"}} Couldn't personalize volume /Volumes/macOS

**Reply by HenryChanhy** [reply]
but on monterey before the upgrade it's worked.

**Comment by HenryChanhy** [comment, marked as answer]
Finally get a work around by install a monterey on a new partition and upgrade the old partition to Ventura again. This time it's worked.
Thanks for the tips to read the advise.
